aws/alarms/lambda/notify_slack/package.json

```diff
@@ -1,15 +1,23 @@
 {
   "name": "notify_slack",
   "version": "1.0.0",
-  "description": "Lambda that notifies multiple channels",
-  "main": "notify_slack.js",
+  "description": "Lambda that notifies Slack and triggers Opsgenie alert",
+  "main": "notify_slack.ts",
+  "type": "module",
   "scripts": {
-    "test": "jest"
+    "prebuild": "rm -rf dist",
+    "test": "node --experimental-vm-modules node_modules/jest/bin/jest.js",
+    "build": "tsc --project tsconfig.json",
+    "build:dev": "tsc --project tsconfig.json --watch",
+    "postbuild": "cp package.json dist/package.json && cp yarn.lock dist/yarn.lock && cd ./dist && yarn install --production"
   },
-  "author": "",
-  "license": "ISC",
+  "license": "MIT",
   "devDependencies": {
+    "@types/aws-lambda": "^8.10.126",
```

Comment:
Some of the packages could use a more recent version

Here is what we have in other lambdas (you will not need all the same packages ofc):
```
{
  "name": "nagware",
  "version": "1.0.0",
  "main": "nagware.js",
  "type": "module",
  "license": "MIT",
  "scripts": {
    "prebuild": "rm -rf dist",
    "build": "tsc --project tsconfig.json",
    "build:dev": "tsc --project tsconfig.json --watch",
    "postbuild": "cp package.json dist/package.json && cp yarn.lock dist/yarn.lock && cd ./dist && yarn install --production"
  },
  "dependencies": {
    "@aws-sdk/client-dynamodb": "3.430.0",
    "@aws-sdk/client-rds-data": "3.430.0",
    "@aws-sdk/client-secrets-manager": "^3.478.0",
    "axios": "^1.6.2",
    "notifications-node-client": "5.2.3",
    "pg": "^8.7.3"
  },
  "devDependencies": {
    "@types/aws-lambda": "^8.10.128",
    "@types/node": "^20.9.4",
    "@types/pg": "^8.10.9",
    "typescript": "^5.3.2"
  }
}

```